openid-4-verifiable-presentations-1_0.md

```diff
@@ -283,10 +281,15 @@ The following additional considerations are given for pre-existing Authorization
 : OPTIONAL. Defined in [@!RFC6749]. The Wallet MAY allow Verifiers to request presentation of Verifiable Credentials by utilizing a pre-defined scope value. See (#request_scope) for more details.
 
 `response_mode`:
-: OPTIONAL. Defined in [@!OAuth.Responses]. This parameter is used (through the new Response Mode `direct_post`) to ask the Wallet to send the response to the Verifier via an HTTPS connection (see (#response_mode_post) for more details). It is also used to request signing and encrypting (see (#jarm) for more details). If the parameter is not present, the default value is `fragment`. 
+: OPTIONAL. Defined in [@!OAuth.Responses]. This parameter is used (through the new Response Mode `direct_post`) to ask the Wallet to send the response to the Verifier via an HTTPS connection (see (#response_mode_post) for more details). It is also used to request signing and encrypting (see (#jarm) for more details). If the parameter is not present, the default value is `fragment`.
 
-The following is a non-normative example of an Authorization Request: 
+The Verifier MAY send an Authorization Request using either of these 3 options:
+1. Passing as URL with encoded parameters
+2. Passing a request object as value
+3. Passing a request object by reference
```

Comment:
The formatting isn't working here, the list is coming out all on the same line:

![Screenshot 2024-07-23 at 21 37 49](https://github.com/user-attachments/assets/fcca1576-0512-43a6-9928-82340b5daedc)

I need it needs blank lines either side:

```suggestion

1. Passing as URL with encoded parameters
2. Passing a request object as value
3. Passing a request object by reference

```

Reply:
Hmm, the i use this VS code extension 
Link: https://marketplace.visualstudio.com/items?itemName=shd101wyy.markdown-preview-enhanced

And the earlier instance rendered fine.
<img width="706" alt="Screenshot 2024-07-23 at 2 04 00 PM" src="https://github.com/user-attachments/assets/c23d854a-16e1-4929-b487-d240040f4a32">

I can update it as per your suggestion.

For future reference can you share the editor/extension you use to render the `md` file?

Reply:
The screenshot was of the official html rendering, which is generated using mmark - I downloaded it from the GitHub action tab, https://github.com/openid/OpenID4VP/pull/218/checks - by clicking on 'artifacts' and then 'output' here:

![Screenshot 2024-07-23 at 22 11 23](https://github.com/user-attachments/assets/be9c748e-ae8b-4922-956b-e6104634aee7)

Reply:
Thanks! will add this to the PR summary